bin/pgaudit_analyze

```diff
@@ -630,9 +636,13 @@ my $bDone = false;
 my $strNextLogFile = nextLogFile($strLogPath);
 
 # Open log file
-open(my $hLog, '>', $strLogOutFile)
+open(my $hLog, '>>', $strLogOutFile)
     or confess "unable to open pgAudit Analyze log file $strLogOutFile: $!";
 
+# always redirect stdout to logfile
+open (STDOUT, '>&', $hLog)
+    or confess "Couldn't close standard output and redirect to logfile: $!";
+
 # Daemonize the process
```

Comment:
Why would you like to make this change?

Reply:
the issue is when run as a systemd process without the --daemon flag all of the  `print` statements do not make it into the logfile
I will also make this its own PR